MAUI/LinearGauge/scale.md

```diff
@@ -0,0 +1,362 @@
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <gauge:SfLinearGauge>
+                <gauge:SfLinearGauge.LineStyle>
+                    <gauge:LinearLineStyle Thickness="20" CornerStyle="BothCurve"/>
+                </gauge:SfLinearGauge.LineStyle>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.LineStyle.Thickness = 20;
+		gauge.LineStyle.CornerStyle = CornerStyle.BothCurve;
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change axis track edge style](images/axis/axis_corner_style.PNG)
+
+Also [`CornerRadius`]() allows to set custom corner style for linear scale. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <gauge:SfLinearGauge>
+                 <gauge:SfLinearGauge.LineStyle>
+                    <gauge:LinearLineStyle Thickness="20" CornerRadius="0,10,0,10"/>
+                </gauge:SfLinearGauge.LineStyle>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.LineStyle.Thickness = 20;
+		gauge.LineStyle.CornerRadius = new Thickness(0, 10, 0, 10);
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change axis track edge style](images/axis/axis_corner_radius.PNG)
+
+N> If you set both [`CornerStyle`]() and [`CornerRadius`]() in the [`LineStyle`](), [`CornerStyle`]() will be first priority for scale. 
+
+## Inverse the axis
+
+The direction of linear gauge axis can be customized by the [`IsInversed`]() property.
+When the [`IsInversed`]() property is true, the axis can be placed in an inverse direction. The default value of the [`IsInversed`]() property is false.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <gauge:SfLinearGauge IsInversed="True">
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.IsInversed = true;
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+You can see that the axis values are displayed from 100 to 0 as the axis track is inversed.
+
+![Inverse linear gauge for axis](images/axis/axis_inversed.PNG)
+
+## Change axis track visibility
+
+You can hide the axis track by setting the [`ShowLine`]() property to false. The default value of this property is true.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <gauge:SfLinearGauge ShowLine="False">
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.ShowLine = false;
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Hide linear gauge axis track](images/axis/hide_axis_track.PNG)
+
+## Customize axis values
+
+Linear gauge allows you to display a set of values along with a custom axis based on your business logic by using the [`GenerateVisibleLabels`](),[`ValueToFactor`]() and [`FactorToValue`]() callbacks.
```

Comment:
[`GenerateVisibleLabels`](),[`ValueToFactor`](), and [`FactorToValue`]() callbacks.